src/private/LoadConfiguration.ps1

```diff
@@ -1,17 +1,29 @@
 function LoadConfiguration {
+    <#
+    .SYNOPSIS
+    Load psake-config.ps1 file
+    .DESCRIPTION
+    Load psake-config.ps1 if present in the directory of the current build script.
+    If that file doesn't exist, load the default psake-config.ps1 file from the module directory.
+    #>
     param(
-        [string] $configdir = $PSScriptRoot
+        [string]$configdir = (Split-Path -Path $PSScriptRoot -Parent)
     )
 
-    $psakeConfigFilePath = (Join-Path $configdir "psake-config.ps1")
+    $configFilePath  = Join-Path -Path $configdir -ChildPath $script:psakeConfigFile
+    $defaultConfigFilePath = Join-Path -Path (Split-Path -Path $PSScriptRoot -Parent) -ChildPath $script:psakeConfigFile
 
-    if (test-path $psakeConfigFilePath -pathType Leaf) {
-        try {
-            [System.Diagnostics.CodeAnalysis.SuppressMessage('PSUseDeclaredVarsMoreThanAssigments', '')]
-            $config = GetCurrentConfigurationOrDefault
-            . $psakeConfigFilePath
-        } catch {
-            throw "Error Loading Configuration from psake-config.ps1: " + $_
-        }
+    if (Test-Path -Path $configFilePath -PathType Leaf) {
+        $configFileToLoad = $configFilePath
+    } elseIf (Test-Path -Path $defaultConfigFilePath -PathType Leaf) {
+        $configFileToLoad = $defaultConfigFilePath
+    }
+

```

Comment:
Should we add an `else` clause here that throws a message like "Cannot find config file" or something to that effect?  It's just in this case, with no throw, we'd proceed to the `. $configFileToLoad` when `$configFileToLoad` would be $null. 

Also, for the `Test-Path` I'd recommend using `-LiteralPath` instead of `-Path`, just in case the current dir has `[]` somewhere in the path.

Reply:
Otherwise LGTM

Reply:
Good catch 👍 I'll make the changes